components/datagrid/FooterRow.js

```diff
@@ -0,0 +1,33 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Cell from './Cell';
+import Row from './Row';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class FooterRow extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    children: PropTypes.any,
+    preserveSelectableSpace: PropTypes.bool,
+    sliceFrom: PropTypes.number,
+    sliceTo: PropTypes.number,
+  };
+
+  render() {
+    const { className, children, sliceFrom, sliceTo, preserveSelectableSpace, ...others } = this.props;
+
+    const childrenArray = Array.isArray(children) ? children : [children];
```

Comment:
Same comment as before

Reply:
This doesn't work because `children` can be either an `object` or an `array`. The spread operator can not handle mixed types.

 :ok: by @driesd on 23/01/2018 15:35:24